app/assets/javascripts/short-form/ShortFormApplicationService.js.coffee.erb

```diff
@@ -309,6 +310,18 @@ ShortFormApplicationService = ($localStorage, $translate, $http, Upload, uuid) -
       Service.preferences["#{fileType}_error"] = true
     ))
 
+  Service.checkSurveyComplete = ->
```

Comment:
@chrisdolendo *ding ding ding* -- correct! More specifically, the landing page for the Review section becomes the summary page (skipping the optional survey) if they've answered all the optional questions. So if `surveyComplete` then when you click next from Income or click in the subnav, it takes you straight to Review summary.